### ISSUE TITLE
 Interface not loading - Specific to Fire OS 7

### ISSUE THREAD

**Issue by Dev-Josue** [wako-app/wako#21]
Wako versions 4.7.8 and above do not load the interface on Fire OS 7 (based on Android 9)
- Different versions of Wako were tried to verify that 4.7.7 is the final version that loads the interface on Fire OS 7
- Wako works on Fire OS 5 up to the latest version, this appears to be version specific (only impacting Fire OS 7)
- Wako works on other Android devices used to test functionality (Android 10)

Steps To Reproduce
-Using an Amazon Fire 9th or 10th generation device, install the Google Play Store and necessary services
-Use Google Play Store to install Wako
-Launch Wako
- User should see a blue screen, Wako fails to load the interface

Issue may be related to missing component in the customized Amazon Fire OS build that became required after version 4.7.7

**Comment by Invictaz** [comment]
It's the same for me on Android 8 Samsung A3 2017

**Comment by JumBay** [comment]
Do you still have the issue? FYI wako has been tested for Android and iOS

**Comment by Dev-Josue** [comment]
Yes unfortunately the issue persists. I updated to the latest version and faced the same blue screen. I will revert to 4.7.7 while the issue is resolved. Please let me know if there is any additional information I can provide to help resolve this issue.

**Comment by JumBay** [comment]
Ok unfortunately it's not something I can test, I don't know what's been modified on Fire OS

**Comment by Dev-Josue** [comment]
Is there any way I can create debug logs for wako?

**Comment by Dev-Josue** [comment]
Also the issue was triggered by the 4.7.8 update so a review of the changes between 4.7.7 and 4.7.8 may help identify the issue.

**Comment by JumBay** [comment]
This is hard to tell cause wako relies on some third party dependencies that have been updated since then. One thing you could try is to update Android WebView from the playstore.

**Comment by Dev-Josue** [comment]
I was missing an update to Android Webview so I was hopeful this would have fixed it but it did not.
Is there an acessible list of dependencies?

**Comment by JumBay** [comment]
well that's mostly web dependencies, but it's weird it has broke after 4.7.7 or the webview that runs on fire os is too old. I'll see what I can do in next release to debug this
